### DISCUSSION TITLE
Port allocation specification in egg

### DISCUSSION THREAD

**Discussion by HardGaming-tech** [pelican-dev/panel#35] [Ideas]
Some eggs by default require multiple eggs. game port, query port, N+1 port, Rcon port. ETC. 

if possible, i would like to see that included as part of the egg so by default the ports necessary are added when the egg is deployed as opposed to having to add in the ports after, or build in some logic to add them via API. 

this can save a lot of time and development efforts.

**Comment by WolfoTheWolf** [comment]
Would make port management much cleaner and easier,. having eggs manage this aspect would 100% be appreciated ❤️

**Comment by lancepioch** [comment]
We are absolutely working on this! This will be a highlight of the next version of Eggs! (older eggs will still work as usual)
